### ISSUE TITLE
Upload image in flutter web

### ISSUE THREAD

**Issue by Meetpatel-09** [flutterchina/dio#1441]
### New Issue Checklist

- [ ] I have searched for a similar issue in the [project](https://github.com/flutterchina/dio/issues) and found none

### Issue Info

| Info                           | Value                                                 |      |
| ------------------------------ | ----------------------------------------------------- | ---- |
| Platform Name                  | e.g. flutter / ios / android                          |      |
| Platform Version               | e.g.  1.5.0 / 12.0 / 9.0                              |      |
| Dio Version                    | e.g.  2.1.0 / 1.0.17                                  |      |
| Android Studio / Xcode Version | e.g. Android Studio 3.3.2 / Xcode 10.2.1              |      |
| Repro rate                     | e.g. all the time (100%) / sometimes x% / only once   |      |
| Repro with our demo prj        | e.g. does it happen with our demo project?            |      |
| Demo project link              | e.g. link to a demo project that highlights the issue |      |

### Issue Description and Steps

Error: Unsupported operation: MultipartFile is only supported where dart:io is available.

**Comment by Meetpatel-09** [comment]
Error: Unsupported operation: MultipartFile is only supported where dart:io is available.

**Comment by jgoyvaerts** [comment]
Your issue doesn't provide any code or details how we could help you. Please make a new issue with proper details and try again.
